helm-chart/todoapp/charts/mysql/templates/secret.yml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ .Chart.Name }}-secrets
+  namespace: {{ .Values.common.namespace }}
+type: Opaque
+data:
+{{- range $k, $v := .Values.mysql.secret }}
+  {{ $k }}: {{ $v | quote }}
+{{- end }}
```

Comment:
Make sure that the `mysql.secret` values in your `values.yaml` file are properly base64 encoded. The `quote` function does not handle base64 encoding, so ensure that the values are already encoded before being templated here.